test/e2e_node/hugepages_test.go

```diff
@@ -154,7 +154,7 @@ func runHugePagesTests(f *framework.Framework) {
 							Limits: v1.ResourceList{
 								v1.ResourceName("cpu"):           resource.MustParse("10m"),
 								v1.ResourceName("memory"):        resource.MustParse("100Mi"),
-								v1.ResourceName("hugepages-2Mi"): resource.MustParse("50Mi"),
+								v1.ResourceName("hugepages-2Mi"): resource.MustParse("6Mi"),
```

Comment:
Nit: I may be missing something obvious, but how come this is `6Mi` instead of `5Mi`?

Reply:
`5Mi` will work, but imo. it is a bit strange to say that we want 2.5 huge pages (since their size is 2Mi). k8s will not complain, same with the kernel. In reality you can then only use `4Mi`, since the kernel will floor the page count.